config/core/configmaps/gc.yaml

```diff
@@ -60,27 +60,50 @@ data:
     # V2 Garbage Collector Settings
     # ---------------------------------------
     #
-    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # These settings are enabled via the "responsive-revision-gc" feature flag.
     # ALPHA NOTE: This feature is still experimental and under active development.
+    #
+    # Stale
+    #   * Revisions are considered stale if they were created after more than
+    #      "retain-since-create-time" or last referenced by a route more than
+    #      "retain-since-last-active-time" ago.
+    #   * Either timeout may be set to "forever" to turn off time-based GC.
```

Comment:
so 'forever' is "disabled" it seems?

Reply:
sorry they are all "disabled". That was from an earlier version